keylime/db/verifier_db.py

```diff
@@ -62,3 +63,12 @@ class VerifierAllowlist(Base):
     generator = Column(Integer)
     tpm_policy = Column(Text())
     ima_policy = Column(Text().with_variant(Text(429400000), "mysql"))
+
+
+class VerifierMbpolicy(Base):
+    __tablename__ = "mbpolicies"
+    __table_args__ = (schema.UniqueConstraint("name", name="uniq_mbpolicies0name"),)
```

Comment:
Any reason why the name contains a "0" instead of "_"?

Reply:
tbh this looks like a typo we didn't catch. I would use an underscore

Reply:
Sure, I will use underscore. Should we also correct the underscore of `allowlists` with this PR?

Reply:
> Sure, I will use underscore. Should we also correct the underscore of `allowlists` with this PR?

I think we should handle the underscore of `allowlists` separately.

Reply:
> Should we also correct the underscore of allowlists with this PR?

No because it is mostly cosmetic and yet still requires DB migration